credential/signing/jwt.go

```diff
@@ -142,44 +152,73 @@ func ParseVerifiableCredentialFromJWT(token string) (*credential.VerifiableCrede
 		cred.CredentialSubject[credential.VerifiableCredentialIDProperty] = subStr
 	}
 
-	jti, hasJti := parsed.Get(jwt.NotBeforeKey)
-	jtiStr, ok := jti.(string)
-	if hasJti && ok && jtiStr != "" {
-		cred.ID = jtiStr
-	}
+	return parsed, &cred, nil
+}
 
-	return &cred, nil
+// JWTVVPParameters represents additional parameters needed when constructing a JWT VP as opposed to a VP
+type JWTVVPParameters struct {
+	// Audience is a required intended audience of the JWT.
+	Audience string `validate:"required"`
```

Comment:
This is an awesome field btw

Reply:
yeah turns out for JWTs you must have an intended audience. cc: @amika-sq